File: dev-docs/snippets/collections/list-collections-for-document.swift
import JsBaoClient

// List collections that contain a specific document.
func listCollectionsForDocument(
  client: JsBaoClient,
  documentId: String
) async throws {
  // #region example
  let page = try await client.collections.listCollectionsForDocument(
    documentId: documentId,
    options: PaginationOptions(limit: 50)
  )
  // #endregion example
  _ = page.items
}

File: dev-docs/snippets/collections/list-collections-for-document.ts
import type { JsBaoClient } from "js-bao-wss-client";

// List collections that contain a specific document.
export async function listCollectionsForDocument(
  client: JsBaoClient,
  documentId: string,
) {
  // #region example
  const page = await client.collections.listCollectionsForDocument(documentId, {
    limit: 50,
  });
  // #endregion example
  return page.items;
}
